src/runtime/virtcontainers/hypervisor_linux.go

```diff
@@ -37,6 +37,8 @@ func NewHypervisor(hType HypervisorType) (Hypervisor, error) {
 		return &Acrn{}, nil
 	case ClhHypervisor:
 		return &cloudHypervisor{}, nil
+	case DragonballHypervisor:
+		return &mockHypervisor{}, nil
```

Comment:
Is this a copy miss? `mockHypervisor` -> ??

Reply:
@liubin Thank for your review, this is the go version runtime, not the runtime-rs.